### PR TITLE
PageObejcts Updates

### PR DESCRIPTION
**Pull request by IshmaelDi**
Added Elements and methods

### CODE REVIEW

**Review by Mnyinyi1 — APPROVED** [review]
Approved

File: src/test/java/PageObjects/ContactPerson.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        //Validate Checkbox using isDisplayed() and click
+
+        WebElement checkBoxElement = getDriver().findElement(By.cssSelector("label[for='hobbies-checkbox-1']"));
+        boolean isDisplayed = checkBoxElement.isDisplayed();
+
+// performing click operation if element is displayed
+        if (isDisplayed) {
+            checkBoxElement.click();
+        }
+
```
Comment (Madoda96) [inline src/test/java/PageObjects/ContactPerson.java:84]: add assertion if button is not displayed

File: src/test/java/PageObjects/ContactPerson.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+
+
+
+
+    @Step("Click on Continue Button")
+    public void ClickOnContinueButton(){
+        $(By.xpath(ContinueButtonXpath)).click();
```
Comment (Madoda96) [inline src/test/java/PageObjects/ContactPerson.java:93]: add assertion for buttons

File: src/test/java/PageObjects/UnderWriting.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        $(By.xpath(TextAreaXpath)).sendKeys(TxtArea);
+
+
+
+    }
+
+    @Step("Click om Checkbox")
+    public void ClickOnCheckBox(){
+        WebElement checkbox = $(By.xpath(CheckBoxXpath));
+
```
Comment (Madoda96) [inline src/test/java/PageObjects/UnderWriting.java:57]: add action on the method

**Pull request merged**